packages/web3-eth-accounts/src/models/Account.js

```diff
@@ -20,7 +20,7 @@
 import scryptsy from 'scrypt.js';
 import isString from 'lodash/isString';
 import isObject from 'lodash/isObject';
-import {fromPrivate, create, sign, decodeSignature} from 'eth-lib/lib/account'; // TODO: Remove this dependency
+import * as EthLibAccount from 'eth-lib/lib/account'; // TODO: Remove this dependency
```

Comment:
👌 namespacing. Is this functionality redundant with ethereumjs-tx?

Reply:
There is ethereumjs package I could use to replace the eth-lib here too. But I will do this later because other enhancements have a higher priority. Thanks for the hint. I'll write it down on my checklist.